init-tools.sh

```diff
@@ -46,18 +46,18 @@ if [ ! -e $__DOTNET_PATH ]; then
         case $OSName in
             Darwin)
                 OS=OSX
-                __DOTNET_PKG=dotnet-dev-osx-x64
+                __DOTNET_PKG=dotnet-sdk-${__DOTNET_TOOLS_VERSION}-osx-x64
```

Comment:
this changed the executable bit on the .sh file.  We should fix that.

Reply:
See the `100755 → 100644` above.

Reply:
I already submitted the fix for this.

Reply:
Actually I only did it in coreclr :) I will do the same for corefx.